src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,94 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+import { getPostComments, addComment } from '../../api/comments';
+
+export const NewCommentForm = ({ postId, setComments }) => {
+  const [comment, setComment] = useState({
+    postId,
+    name: '',
+    email: '',
+    body: '',
+  });
+
+  const addFieldToComment = (event) => {
+    const { name, value } = event.target;
+
+    setComment(createdComment => ({
+      ...createdComment,
+      [name]: value,
+    }));
+  };
+
+  const clearForm = () => {
+    setComment({
+      postId,
+      name: '',
+      email: '',
+      body: '',
+    });
+  };
+
+  const addNewComment = () => {
+    addComment(comment);
+    getPostComments(postId).then(setComments);
+  };
```

Comment:
```suggestion
  const addNewComment = async() => {
    await addComment(comment);
    getPostComments(postId).then(setComments);
  };
```
I guess it should solve a problem with a twice-click for rendering comments.